packages/flutter/lib/src/material/search.dart

```diff
@@ -111,6 +112,11 @@ abstract class SearchDelegate<T> {
   ///   Widget buildLeading(BuildContext context) => Text("leading");
   ///
   ///   @override
+  ///   PreferredSizeWidget buildBottom(BuildContext context) => PreferredSize(
```

Comment:
This could be slightly more consistent with Flutter's style:
```dart
///   PreferredSizeWidget buildBottom(BuildContext context) {
///     return PreferredSize(
///       preferredSize: Size.fromHeight(56.0),
///       child: Text("bottom"));
///   }
```

Reply:
Updated